### DISCUSSION TITLE
Can I use a different sort of dataset and also what if I don't have a bval , I use . mat images format PATCH2Self

### DISCUSSION THREAD

**Discussion by doro041** [dipy/dipy#3094] [Preprocessing]
## Description
I am dealing with a different type of MRI , which has a similar data type of images - 4D , however  I do not have bval file. Any ideas?

**Comment by doro041** [comment]
for the denoising with patch2self,also I was wondering, but what could be the issue if one cannot see denoising effect?

<img width="1052" alt="Screenshot 2024-03-03 at 22 37 05" src="https://github.com/dipy/dipy/assets/64807351/027c27c0-9ea6-40aa-8a2b-f4fcc1d754a3">

**Comment by skoudoro** [comment]
Hi @doro041,

Thank you for your question. I convert this issue to a discussion.

@ShreyasFadnavis, can you answer to @doro041 ? thanks!
